contracts/reputation/Reputation.sol

```diff
@@ -458,37 +458,6 @@ contract Reputation is MasterCopyNonUpgradable, ConsensusModule {
         );
     }
 
-    /**
-     * @notice Slashes validator.
-     *
-     * @dev Function requires:
-     *          - only consensus can call
-     *          - a validator has staked
-     *          - a validator has not withdrawn
-     *
-     * @param _validator A validator address to slash.
-     */
-    function slash(address _validator)
-        external
-        onlyConsensus
-        hasStaked(_validator)
-        hasNotWithdrawn(_validator)
-        hasNotSlashed(_validator)
```

Comment:
hasNotSlashed modifier is not used anymore. We can removed this modifier.

Reply:
Done

Reply:
resolved